src/main/java/org/dependencytrack/integrations/gitlab/GitLabSyncer.java

```diff
@@ -79,8 +77,9 @@ public boolean isEnabled() {
 
     @Override
     public void synchronize() {
-        // TODO: GitLab API call to get user projects
-        List<String> projects = new ArrayList<>();
+        GitLabClient gitLabClient = new GitLabClient(this, baseURL, this.accessToken);
```

Comment:
I **_think_** the URL value we want will be the one the user sets in the integration settings instead of pulling out of alpine application properties.

```suggestion
        final ConfigProperty gitLabUrl = qm.getConfigProperty(INTEGRATIONS_GROUP, GITLAB_URL.getPropertyName());
        gitLabClient = new GitLabClient(this, URI.create(gitLabUrl.getPropertyValue()), accessToken);
```